crates/specta/tests/ts_rs/field_override.rs

```diff
@@ -0,0 +1,13 @@
+use specta::{ts::ts_inline, Type};
+
+#[derive(Type)]
+struct Override {
+    a: i32,
+    #[specta(type_as = String)]
+    b: i32,
+}
+
+#[test]
+fn test() {
```

Comment:
The `ts_rs` tests are meant to be close reflections of `ts-rs`'s actual tests - the relevant one for type overrides is [here](https://github.com/Aleph-Alpha/ts-rs/blob/f2dbbc8c10ba7071acddbe69480edfd7cfe0fbc7/ts-rs/tests/type_override.rs). Just change the file name and make the `ts-rs` test work with Specta and it's fine.